community-content/pytorch_text_classification_using_vertex_sdk_and_gcloud/pytorch-text-classification-vertex-ai-train-tune-deploy.ipynb

```diff
@@ -1834,7 +1834,7 @@
         "%%writefile ./custom_container/Dockerfile\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    # BUCKET_NAME: Change to your bucket name
</pre><p>There are 2 comments here about changing the bucket name. Is that correct, or is bucket name a parameter into this script?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='185'/>

Reply:
<p>Bucket name is already set earlier in the notebook and is expected as a parameter to this cell that executes gcloud cli to trigger a hp tuning job. However, I have seen a customer copying the cell directly into a bash file. Just wanted to make sure they pass the bucket name. I am fine to change the comment line to: <code>BUCKET_NAME is a required parameter to run the cell.</code>  and open for any suggestions</p>

Reply:
I agree with your suggestion